### DISCUSSION TITLE
Assign two or more trigger fields

### DISCUSSION THREAD

**Discussion by man-tax** [Budibase/budibase#5525] [Help] [answered]
Hey everyone,
quick question please. What's the right syntax to assign two trigger fields to the Webhook input variables? There's only {{ trigger.fields }} to select and it includes the URL (first trigger field). Thanks! Sebastian

![image](https://user-images.githubusercontent.com/104101494/164544209-434647f3-0088-43d5-ac42-f667ecc8fe38.png)

**Comment by melohagan** [comment, marked as answer]
Hey @man-tax 

You'll need to add in the field names. E.g. `{{ trigger.fields.URL }}`

**Reply by man-tax** [reply]
Hi melohagan, thanks a lot! Working fine.

Hope you don't mind I'm raising another one...!

Having a similiar problem when receiving data under my webhook. I only see {{ trigger.body }} to bind, and tried variations of it. Also I've sent a request to the schema url (as requested during set up of the webhook), but nothing happens (and don't know what's supposed to happen actually). {{ trigger.body }} returns what's I've put in as request content in Integromat, but I have an additional variable and not sure how I get it? Not really sure how that's supposed to work?

![image](https://user-images.githubusercontent.com/104101494/164763712-5b41e09d-14c8-414a-b14d-4649d5c74c67.png)


Thanks!
Sebastian

**Reply by man-tax** [reply]
Found a different solution for my issue, but still the POST to the schema URL doesn't work for me (or I'm doing it wrong).
